CoreFoundation/RunLoop.subproj/CFSocket.c

```diff
@@ -2174,9 +2175,9 @@ manageSelectError()
 
 static void *__CFSocketManager(void * arg)
 {
-#if (TARGET_OS_LINUX && !TARGET_OS_CYGWIN) || TARGET_OS_BSD
+#if (TARGET_OS_LINUX && !TARGET_OS_CYGWIN)
     pthread_setname_np(pthread_self(), "com.apple.CFSocket.private");
-#elif TARGET_OS_CYGWIN
+#elif TARGET_OS_CYGWIN || TARGET_OS_BSD
```

Comment:
You didn't introduce this, but, would you mind rewriting this as:
```
#elif !TARGET_OS_CYGWIN && !TARGET_OS_BSD
```
and drop the empty clause instead?

Reply:
Done.